File: Competitive Programming/DAY-6/free_bytes.py
f=int(input("Enter the no of bytes avaiable"))
u=int(input("Enter the no of bytes used"))
o=int(input("Enter the deleted file number of bytes"))
n=int(input("Enter the new file no of bytes"))
freeSize=(f+o)-n
if f+o < n:
    print("Size Full")
else:
    print("Free bytes avaiable is ",freeSize)

File: Competitive Programming/DAY-6/free_bytes.cpp

#include <iostream>

using namespace std;

int main()
{
    int f,n,u,o;
    cin>>f>>u>>o>>n;
    int freesize=(f+o)-n;
    if ((f+o) < n)
    cout<<"FULL"<<endl;
    else
    cout<<"size free is "<<freesize;
    return 0;
}
